### DISCUSSION TITLE
Compatibility with locally installed Carla 0.9.15

### DISCUSSION THREAD

**Discussion by harsha1597** [guardstrikelab/carla_apollo_bridge#190] [Q&A] [answered]
Hi,

I need to upgrade my Carla packages to 0.9.15 for another project. Does the Apollo-Carla bridge work with locally installed Carla 0.9.15 ?

Thanks

**Comment by TayYim** [comment, marked as answer]
I tried with 0.9.15, it works fine

**Reply by harsha1597** [reply]
Did you also try with Apollo v9 ?

**Reply by TayYim** [reply]
Yep, Apollo 9.0.0 works fine with Carla 0.9.15 through this bridge.  
I mean, the vehicle runs as expected in Plan&Control mode at least. I'm not sure whether perception modules are working fine.
